.devcontainer/devcontainer.json

```diff
@@ -1,15 +1,21 @@
 {
-	"name": "template.py",
-	"build": {
-		"dockerfile": "Dockerfile",
-		"context": "..",
-		"args": {
-			// Update 'VARIANT' to pick a Python version: 3, 3.10, 3.9, 3.8, 3.7, 3.6
-			// Append -bullseye or -buster to pin to an OS version.
-			// Use -bullseye variants on local on arm64/Apple Silicon.
-			"VARIANT": "3"
-		}
+	"name": "template.py.simple",
+	"image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+
+	"features": {
```

Comment:
// Features to add to the dev container. More info: https://containers.dev/features.